front/app/translations/admin/en-CA.json

```diff
@@ -634,7 +590,6 @@
   "app.containers.AdminPage.DashboardPage.selectedProject": "current project filter",
   "app.containers.AdminPage.DashboardPage.selectedTopic": "current tag filter",
   "app.containers.AdminPage.DashboardPage.subtitleDashboard": "Discover what's happening on your platform.",
-  "app.containers.AdminPage.DashboardPage.tabInsights": "Insights",
```

Comment:
I'm guessing you're pretty confident about this one, so I'll stop commenting on every occurrence ;-)

Reply:
These changes happen automatically for all languages once a string is no longer needed ;)